common/djangoapps/student/admin.py

```diff
@@ -155,7 +155,7 @@ class CourseEnrollmentAdmin(admin.ModelAdmin):
     list_display = ('id', 'course_id', 'mode', 'user', 'is_active',)
     list_filter = ('mode', 'is_active',)
     raw_id_fields = ('user',)
-    search_fields = ('course_id', 'mode', 'user__username',)
+    search_fields = ('course__id', 'mode', 'user__username',)
```

Comment:
While this suppresses the original bug, I'm not sure it's useful to search on that ID (which has a very different meaning than `CourseEnrollment.course_id`).

Reply:
@edx/educator-devs Any opinion on what's the right course of action here?

Reply:
@gsong this fix is correct.  A recent [PR](https://github.com/edx/edx-platform/pull/15878) changed the `CourseEnrollment` model to add a new foreign key relationship to the `CourseOverview` model.  So this change will now access the `CourseKey` value of the `CourseOverview`'s `id` field.